config/environments/development.rb

```diff
@@ -48,9 +48,9 @@
       user_name: ENV.fetch('SMTP_USERNAME', nil),
       password: ENV.fetch('SMTP_PASSWORD', nil),
       authentication: ENV.fetch('SMTP_AUTH', nil),
-      enable_starttls_auto: ENV.fetch('SMTP_STARTTLS_AUTO', true),
-      enable_starttls: ENV.fetch('SMTP_STARTTLS', false),
-      tls: ENV.fetch('SMTP_TLS', 'false') != 'false',
+      enable_starttls_auto: ActiveModel::Type::Boolean.new.cast(ENV.fetch('SMTP_STARTTLS_AUTO', 'true')),
```

Comment:
Just a note, any none explicit none emptoy value of `'true'|'false'` will always be `true`,

Reply:
Yep - expected